tests/integration/conftest.py

```diff
@@ -602,3 +602,30 @@ def create_file_structure_function(get_files_list):
     yield
 
     delete_file_structure(get_files_list)
+
+
+@pytest.fixture(scope="module")
+def configure_local_internal_options_module(request):
+    """Fixture to configure the local internal options file.
+
+    It uses the test variable local_internal_options. This should be
+    a dictionary wich keys and values corresponds to the internal option configuration, For example:
+    local_internal_options = ['monitord.rotate_log': '0', 'syscheck.debug': '0' ]
+    """
+    try:
+        local_internal_options = getattr(request.module, 'local_internal_options')
+    except AttributeError as local_internal_configuration_not_set:
+        logger.error('Configure Local Internal Options: Error - local_internal_options is not set')
+        raise local_internal_configuration_not_set
+
+    backup_local_internal_options = conf.get_local_internal_options_dict()
+
+    logger.error('Configure Local Internal Options: DEBUG - Set local_internal_option to '  +
+                    f"{str(local_internal_options)}")
```

Comment:
This logger lever should be `info` or `debug`

Reply:
Done 87039c12c10a6430991b78dae1796a53496c1600